mitodl/open-discussions
# Add can_remove field to serializers

#### What are the relevant tickets?
Part of #1004

#### What's this PR do?
Adds a field to the moderators REST API showing who can and can't be removed by the logged in user (who must also be a moderator)

#### How should this be manually tested?
View `/api/v0/channels/<channel_name>/moderators/` and verify that the `can_remove` field is True for you and other moderators, and False for older moderators
